secure-api-gateway-ob-uk-rcs-server/src/main/java/com/forgerock/sapi/gateway/ob/uk/rcs/server/RCSServerApplication.java

```diff
@@ -26,9 +26,9 @@
         }
 )
 @SpringBootApplication
-public class RcsApplication {
+public class RCSServerApplication {
```

Comment:
I think RcsApplication was better.

The S in RCS stands for server.

Not a big deal so I'll approve and leave it up to you.